apisix/plugins/proxy-mirror.lua

```diff
@@ -53,7 +60,18 @@ function _M.rewrite(conf, ctx)
     core.log.info("proxy mirror plugin rewrite phase, conf: ", core.json.delay_encode(conf))
 
     ctx.var.upstream_host = ctx.var.host
-    ctx.var.upstream_mirror_host = conf.host
+
+    if not conf.percentage or conf.percentage == 1 then
+        ctx.var.upstream_mirror_host = conf.host
+    else
+        math.randomseed(socket.gettime())
```

Comment:
`randomseed` was already called in `http_init_worker`.

Reply:
thanks~ done~